conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalytics.java

```diff
@@ -167,14 +217,14 @@ public void reportPlaybackDeficiency(String message, Client.ErrorSeverity severi
      * Sends a custom deficiency event during playback to Conviva's Player Insight. If no session is active it will NOT
      * create one.
      *
-     * @param message Message which will be send to conviva
-     * @param severity One of FATAL or WARNING
+     * @param message    Message which will be send to conviva
+     * @param severity   One of FATAL or WARNING
      * @param endSession Boolean flag if session should be closed after reporting the deficiency
-    */
+     */
     public void reportPlaybackDeficiency(String message,
                                          Client.ErrorSeverity severity,
```

Comment:
Not a change in this PR, but why is each of the parameter of this method in an extra line?

Reply:
Let's keep it in the other PR (#12)